tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/annotations/AnnotationRuntimeTest.java

```diff
@@ -45,6 +45,7 @@
 public class AnnotationRuntimeTest {
 
     private CompileResult resultAccessNegative;
+    private final CompileResult resultOne = BCompileUtil.compile("test-src/annotations/annot_access.bal");
```

Comment:
Not sure if this is a good idea. This object may be retained until the complete test suite is complete.

Reply:
Moved this out because it was used in `testAnnotAccess` which uses a data provider, resulting in the compilation happening over and over again for each test.